feeds/wlan-ap/opensync/files/etc/hotplug.d/net/60-dynamic-vlan

```diff
@@ -1,5 +1,20 @@
+#!/bin/sh
+. /usr/share/libubox/jshn.sh
 [ "$ACTION" = "add" ] || exit 0
 
+json_init
+json_load "$(cat /etc/board.json)"
+json_select network
+	json_select "wan"
+		json_get_vars ifname
+	json_select ..
+json_select ..
+
+[ -n "$ifname" ] || {
+	ifname=$(uci get network.wan.ifname)
+	ifname=${ifname%% *}
+}
+
```

Comment:
Same comment as before: will network_find_wan() /lib/functions/network.sh accomplish the same thing?

Reply:
It uses ubus call network.interface dump internally, I don't see any info containing the ethX interfaces in it.